app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,290 @@
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag 'metrics_page', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'practice_editor_utilities', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
```

Comment:
You can remove this if there's nothing going inside the block.

Reply:
added _metrics partial for js in _metrics.js.erb